src/main/java/com/robinpowered/sdk/RobinApi.java

```diff
@@ -83,6 +83,10 @@ public RobinApi(Credential credential) {
      * @param httpClient An HTTP client for performing network requests.
      */
     public RobinApi(Credential credential, Client httpClient) {
+        if (credential != null && credential.getValue() == null) {
```

Comment:
For the purposes of clients being able to de-authenticate their `RobinApi` client, I think it would be appropriate to allow having a null credential.

If a class wanted to keep reference to its `RobinApi` client, yet goes through a cycle of de-authenticating and re-authenticating, say in a logout/login scenario, we wouldn't want to throw an exception here.